src/server/backend-lambda.ts

```diff
@@ -0,0 +1,5 @@
+import { ApolloServer } from 'apollo-server-lambda';
+import { apolloServerConfig } from './apollo-server-config';
+
+const server = new ApolloServer(apolloServerConfig);
+export const handler = server.createHandler();
```

Comment:
Could you describe briefly what an authorizer would look like for this lambda?

Reply:
I initially thought about simply creating an authorizer lambda and plugging it into the API gateway, but with just this approach, it wouldn't be possible to have public queries/mutations as well, so it would be necessary to do something more... 

I guess I would probably do part of the auth work inside the authorizer lambda, and then do the rest inside my backend lambda (like checking if the query/mutation that the user wants to consume is public or not).

Reply:
To be honest, I am starting to think that an authorizer lambda may not be so useful in this scenario, so I guess I would actually do everything inside my backend lambda: I would parse and validate the user token when building the context, put the useful info inside it, and let the resolvers decide if the user is good to go or not.